opt/group_load.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+if [[ $# -ne 1 ]]
+    then
+        echo "usage: $0 config_file"
+        echo "  config_file: path to the file containing environment variables for the patient group to be used"
+        exit 2
+fi
+
+if [[ ! -f $1 ]]
+    then
+        echo "invalid file path: $1"
+        exit 2
+fi
+
+source "$1"
+
+list_idx=0
+for i in {00001..00300}
```

Comment:
Mostly, thinking out loud: 
Could we give a better name to `i`? I would like it to make it clear what the range of numbers is, if possible. The answer could be that it is just a range and in that case `i` is ok. Just that a better name would be easier to `grep` etc.

Reply:
I agree with Aman's comments, otherwise a great improvement; thanks!

Reply:
I'm not really sure what to change `i` to. It's just there because I noticed that in the original scripts there were `.vcf.gz.tbi` and `.vcf.gz` files which were in the format of `group1_{i}` (for group1) where `i` went from 1 to 300.

Reply:
I propose `vcf_filename_ord` or `vcf_filename_index`. What do you think?